db/objects/impact/V.0.1.0.70__impact-food-availability.sql

```diff
@@ -0,0 +1,41 @@
+CREATE TABLE IMPACT_FOOD_AVAILABILITY (
+	ssp                 		text REFERENCES IMPACT_SCENARIO (ID)
+	, comdty            		text REFERENCES IMPACT_COMMODITY (ID)
+	, cty       				text REFERENCES COUNTRY (ID)
+	, yr          				smallint
```

Comment:
We can use full words, the letter shortage has ended :P 

Also, I've noticed that the code is mixing tabs and spaces, you may want to check your editor settings

```suggestion
	, commodity                 text REFERENCES IMPACT_COMMODITY (ID)
	, country                   text REFERENCES COUNTRY (ID)
	, year                      smallint
```